Misc/NEWS.d/next/Core and Builtins/2019-09-02-16-17-42.bpo-36946._lAuSR.rst

```diff
@@ -0,0 +1 @@
+A complement to fix possible signed integer overflow when handling slices.
```

Comment:
Please add "Patch by ...".

And maybe remove "A complement"? This does not say anything to the average reader of the changelog.